src/services/template/seed.ts

```diff
@@ -0,0 +1,70 @@
+import "reflect-metadata"
+import { createConnection } from 'typeorm';
+import { User } from './entity/User';
+
+createConnection().then(async (connection) => {
+    const userRepository = connection.getRepository(User);
+
+    const userOne = new User();
+    userOne.name = 'Mahmood Shilleh';
+    userOne.email = 'Shilleh@gmail.com';
+    userOne.password = 'Shilleh@gmail.com';
+    userRepository.save(userOne);
```

Comment:
Try replacing this with `User.insert` in the same way that I use it in the UserResolvers.ts file on line 122. It seems like that would be much cleaner that way

Reply:
And this should be applied to all of the users you are creating.